webpack.config.js

```diff
@@ -16,7 +16,7 @@ const MomentLocalesPlugin = require('moment-locales-webpack-plugin');
 const devMode = process.env.NODE_ENV !== 'production';
 
 const pluginSystem = [
-  new CleanWebpackPlugin(['web/build']),
+  new CleanWebpackPlugin({ cleanAfterEveryBuildPatterns: ['web/build'] }),
```

Comment:
I get failures using `npm run watch` related to `Error: ENOTEMPTY...` (see screenshot) and this change fixes it:

```suggestion
  new CleanWebpackPlugin({ cleanOnceBeforeBuildPatterns: ['./web/build'] }),
```
![image](https://user-images.githubusercontent.com/24417194/62558925-f559c280-b847-11e9-957e-e60db0b3ab7a.png)

Reply:
`cleanOnceBeforeBuildPatterns` does not include rebuilds (watch mode), so I'm not sure if using that method will have the desired effect.